vowpalwabbit/lda_core.cc

```diff
@@ -923,7 +923,7 @@ void learn_batch(lda &l)
   float additional = (float)(l.all->length()) * l.lda_rho;
   for (size_t i = 0; i < l.all->lda; i++) l.digammas.push_back(l.digamma(l.total_lambda[i] + additional));
 
-  uint64_t last_weight_index = -1;
+  auto last_weight_index = static_cast<uint64_t>(-1);
```

Comment:
Does it make sense to use the max value here instead like in cs_active?

Reply:
I was unsure since the [cppreference](https://en.cppreference.com/w/cpp/types/numeric_limits/max) doesn't specify the `uint64_t` type but I think it shouldn't be a problem to use the numeric_limits